webapp/src/templates/partials/contacts_deceased.html

```diff
@@ -20,7 +20,7 @@ <h3 translate>contact.deceased.title</h3>
             </div>
           </div>
           <ul>
-            <mm-content-row ng-repeat="child in $ctrl.selected.children.deceased"
+            <mm-content-row ng-repeat="child in $ctrl.selected.children" ng-if="child.deceased"
```

Comment:
`$ctrl.selected.children` are grouped now, so you'd need to:
```
<div ng-repeat="group in $ctrl.selected.children" ng-if="group.type.person && group.deceasedCount">
...
<mm-content-row ng-repeat="child in group.contacts" ng-if="child.deceased">
```

Reply:
Done. Good spotting.